### PR TITLE
external/webclient : fix bad request parameter

### PR DESCRIPTION
**Pull request by JeonginKim**
fix bad request bug
add debug message

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: external/webclient/webclient.c [hunk 1]
```diff
@@ -1161,13 +1161,16 @@ static pthread_addr_t wget_base(void *arg)
 		goto errout_before_init;
 	}
 
+	ndbg("Length Info: sndlen : %d buf_len : %d param->buglen : %d\n", strlen(ws.buffer), strlen(param->buffer), param->buflen);
```
Comment (sunghan-chang) [inline external/webclient/webclient.c]: If content is an information, you should use nvdbg, not ndbg. The ndbg is for error.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by JeonginKim** [comment]
I changed  ndbg -> nvdbg.

**Pull request merged**